app/controllers/feedback_controller.rb

```diff
@@ -1,35 +1,49 @@
 class FeedbackController < ApplicationController
+  include OrganizationScopable
+
   skip_before_action :authenticate_user!
-  skip_verify_authorized only: %i[new create]
-  layout :set_layout, only: %i[new create]
+  skip_verify_authorized only: %i[new create show]
+  layout :set_layout, only: %i[new]
+
+  def show
+  end
 
   def new
-    @contact = Contact.new
+    @feedback = Feedback.new
   end
 
   def create
-    @contact = Contact.new(contact_params)
-    if @contact.valid?
-      FeedbackMailer.with(contact_params).send_message.deliver_later
-      redirect_to root_path, notice: I18n.t("contacts.create.success")
+    @feedback = Feedback.new(feedback_params)
+
+    if @feedback.valid?
+      FeedbackMailer.with(feedback_params).send_message.deliver_later
```

Comment:
Let's use `deliver_now` for the time being, until we set up background processing.